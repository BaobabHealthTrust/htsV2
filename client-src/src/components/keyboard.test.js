import React from 'react';
import ReactDOM from 'react-dom';
import Keyboard from './keyboard';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import Enzyme, {
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
});

describe('Component: Keyboard', () => {

    it('should match its empty snapshot', () => {
        const tree = renderer.create( 
            <Keyboard />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })

})