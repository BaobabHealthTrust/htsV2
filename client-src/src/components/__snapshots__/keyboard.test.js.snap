// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Keyboard should match its empty snapshot 1`] = `
<div>
  <div
    id="row_1"
  >
    <button
      className="blue"
      id="qwe_q"
      onMouseDown={[Function]}
      style={undefined}
    >
      Q
    </button>
    <button
      className="blue"
      id="qwe_w"
      onMouseDown={[Function]}
      style={undefined}
    >
      W
    </button>
    <button
      className="blue"
      id="qwe_e"
      onMouseDown={[Function]}
      style={undefined}
    >
      E
    </button>
    <button
      className="blue"
      id="qwe_r"
      onMouseDown={[Function]}
      style={undefined}
    >
      R
    </button>
    <button
      className="blue"
      id="qwe_t"
      onMouseDown={[Function]}
      style={undefined}
    >
      T
    </button>
    <button
      className="blue"
      id="qwe_y"
      onMouseDown={[Function]}
      style={undefined}
    >
      Y
    </button>
    <button
      className="blue"
      id="qwe_u"
      onMouseDown={[Function]}
      style={undefined}
    >
      U
    </button>
    <button
      className="blue"
      id="qwe_i"
      onMouseDown={[Function]}
      style={undefined}
    >
      I
    </button>
    <button
      className="blue"
      id="qwe_o"
      onMouseDown={[Function]}
      style={undefined}
    >
      O
    </button>
    <button
      className="blue"
      id="qwe_p"
      onMouseDown={[Function]}
      style={undefined}
    >
      P
    </button>
  </div>
  <div
    id="row_2"
  >
    <button
      className="blue"
      id="qwe_a"
      onMouseDown={[Function]}
      style={undefined}
    >
      A
    </button>
    <button
      className="blue"
      id="qwe_s"
      onMouseDown={[Function]}
      style={undefined}
    >
      S
    </button>
    <button
      className="blue"
      id="qwe_d"
      onMouseDown={[Function]}
      style={undefined}
    >
      D
    </button>
    <button
      className="blue"
      id="qwe_f"
      onMouseDown={[Function]}
      style={undefined}
    >
      F
    </button>
    <button
      className="blue"
      id="qwe_g"
      onMouseDown={[Function]}
      style={undefined}
    >
      G
    </button>
    <button
      className="blue"
      id="qwe_h"
      onMouseDown={[Function]}
      style={undefined}
    >
      H
    </button>
    <button
      className="blue"
      id="qwe_j"
      onMouseDown={[Function]}
      style={undefined}
    >
      J
    </button>
    <button
      className="blue"
      id="qwe_k"
      onMouseDown={[Function]}
      style={undefined}
    >
      K
    </button>
    <button
      className="blue"
      id="qwe_l"
      onMouseDown={[Function]}
      style={undefined}
    >
      L
    </button>
  </div>
  <div
    id="row_3"
  >
    <button
      className="blue"
      id="qwe_z"
      onMouseDown={[Function]}
      style={undefined}
    >
      Z
    </button>
    <button
      className="blue"
      id="qwe_x"
      onMouseDown={[Function]}
      style={undefined}
    >
      X
    </button>
    <button
      className="blue"
      id="qwe_c"
      onMouseDown={[Function]}
      style={undefined}
    >
      C
    </button>
    <button
      className="blue"
      id="qwe_v"
      onMouseDown={[Function]}
      style={undefined}
    >
      V
    </button>
    <button
      className="blue"
      id="qwe_b"
      onMouseDown={[Function]}
      style={undefined}
    >
      B
    </button>
    <button
      className="blue"
      id="qwe_n"
      onMouseDown={[Function]}
      style={undefined}
    >
      N
    </button>
    <button
      className="blue"
      id="qwe_m"
      onMouseDown={[Function]}
      style={undefined}
    >
      M
    </button>
    <button
      className="blue"
      id="qwe_-"
      onMouseDown={[Function]}
      style={undefined}
    >
      -
    </button>
    <button
      className="blue"
      id="qwe_cap"
      onMouseDown={[Function]}
      style={undefined}
    >
      cap
    </button>
  </div>
  <div
    id="row_4"
  >
    <button
      className="blue"
      id="qwe_del"
      onMouseDown={[Function]}
      style={undefined}
    >
      del
    </button>
    <button
      className="blue"
      id="qwe_clear"
      onMouseDown={[Function]}
      style={undefined}
    >
      clear
    </button>
    <button
      className="blue"
      id="qwe_:"
      onMouseDown={[Function]}
      style={undefined}
    >
      :
    </button>
    <button
      className="blue"
      id="qwe_."
      onMouseDown={[Function]}
      style={undefined}
    >
      .
    </button>
    <button
      className="blue"
      id="qwe_/"
      onMouseDown={[Function]}
      style={undefined}
    >
      /
    </button>
    <button
      className="blue"
      id="qwe_space"
      onMouseDown={[Function]}
      style={undefined}
    >
      space
    </button>
    <button
      className="blue"
      id="qwe_num"
      onMouseDown={[Function]}
      style={undefined}
    >
      num
    </button>
    <button
      className="blue"
      id="qwe_abc"
      onMouseDown={[Function]}
      style={undefined}
    >
      abc
    </button>
  </div>
</div>
`;
